client/src/pages/List.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from "react";
+import { DragDropContext, Droppable, Draggable } from "react-beautiful-dnd";
+import "../List.css";
+
+function List() {
+  const examples = ["math", "english", "science", "history"];
+
+  const [list, setList] = useState(examples);
+
+  function handleOnDragEnd(res) {
+    // console.log(res);
+    if (!res.destination) return;
+    const items = Array.from(list);
+    const [reorderedItem] = items.splice(res.source.index, 1);
+    items.splice(res.destination.index, 0, reorderedItem);
+    setList(items);
+  }
+
+  return (
+    <div className="list-container">
+      <h1>List</h1>
```

Comment:
Try to generally use the MaterialUI components whenever possible, in this case `<Typography variant="h1">`